backend/protos/mgmt/v1alpha1/user_account.proto

```diff
@@ -178,7 +195,7 @@ message GetAccountStatusResponse {
   optional uint64 allowed_record_count = 2;
 
   // The current subscription status determined by the billing system.
-  int32 subscription_status = 3;
+  BillingStatus subscription_status = 3;
```

Comment:
this techncially breaks the api but it's still an int so it is fine (and unused...)